recipes/hidefix/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "hidefix" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hidefix-{{ version }}.tar.gz
+  sha256: 9ac6c81783e1d8c401c4c5bd2d2df1f6f85bc91de32889d54acf62506bbd6910
+
+build:
+  number: 0
+  script:
+    - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
+    - {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - cmake
+    - make
+    - pkg-config
+    - python                              # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - maturin                             # [build_platform != target_platform]
+    - {{ compiler('c') }}                 # [linux]
+    - {{ compiler('m2w64_c') }}           # [win]
```

Comment:
Hmm, looks like Windows builds are failing at https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=762255&view=logs&j=240f1fee-52bc-5498-a14a-8361bde76ba0&t=2fb5b0a7-737c-5f5c-8f3f-f6db6174bacf&l=1746

```python-traceback
  error: failed to run custom build command for `pyo3-build-config v0.17.3`

  Caused by:
    process didn't exit successfully: `C:\bld\hidefix_1692082449997\work\target\release\build\pyo3-build-config-9474a066d1550e39\build-script-build` (exit code: 1)
    --- stdout
    cargo:rerun-if-env-changed=PYO3_CONFIG_FILE
    cargo:rerun-if-env-changed=PYO3_NO_PYTHON
    cargo:rerun-if-env-changed=PYO3_PYTHON
    cargo:rerun-if-env-changed=VIRTUAL_ENV
    cargo:rerun-if-env-changed=CONDA_PREFIX

    --- stderr
    error: failed to run the Python interpreter at C:\bld\hidefix_1692082449997\_build_env\python.exe: The system cannot find the file specified. (os error 2)
  warning: build failed, waiting for other jobs to finish...
  💥 maturin failed
    Caused by: Failed to build a native library through cargo
    Caused by: Cargo build finished with "exit code: 101": `"cargo" "rustc" "--release" "--features" "static" "--features" "python" "--features" "extension-module" "--features" "fast-index" "--manifest-path" "C:\\bld\\hidefix_1692082449997\\work\\Cargo.toml" "--message-format" "json" "--lib" "--crate-type" "cdylib"`
  Error: command ['maturin', 'pep517', 'build-wheel', '-i', 'C:\\bld\\hidefix_1692082449997\\_h_env\\python.exe', '--compatibility', 'off'] returned non-zero exit status 1
  error: subprocess-exited-with-error
  
  Building wheel for hidefix (pyproject.toml) did not run successfully.
  exit code: 1
  
  See above for output.
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  full command: 'C:\bld\hidefix_1692082449997\_h_env\python.exe' 'C:\bld\hidefix_1692082449997\_h_env\lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py' build_wheel 'C:\Users\VSSADM~1\AppData\Local\Temp\tmpuhibrxi7'
  cwd: C:\bld\hidefix_1692082449997\work
  Building wheel for hidefix (pyproject.toml): finished with status 'error'
  ERROR: Failed building wheel for hidefix
Failed to build hidefix
ERROR: Could not build wheels for hidefix, which is required to install pyproject.toml-based projects
Exception information:
Traceback (most recent call last):
  File "%PREFIX%\lib\site-packages\pip\_internal\cli\base_command.py", line 180, in exc_logging_wrapper
    status = run_func(*args)
  File "%PREFIX%\lib\site-packages\pip\_internal\cli\req_command.py", line 248, in wrapper
    return func(self, options, args)
  File "%PREFIX%\lib\site-packages\pip\_internal\commands\install.py", line 429, in run
    raise InstallationError(
pip._internal.exceptions.InstallationError: Could not build wheels for hidefix, which is required to install pyproject.toml-based projects
Removed build tracker: 'C:\\Users\\VssAdministrator\\AppData\\Local\\Temp\\pip-build-tracker-rq1v8b8t'
Traceback (most recent call last):
  File "D:\a\1\s\.ci_support\build_all.py", line 218, in <module>
    build_all(os.path.join(root_dir, "recipes"), args.arch)
  File "D:\a\1\s\.ci_support\build_all.py", line 118, in build_all
    build_folders(recipes_dir, folders, arch, channel_urls)
  File "D:\a\1\s\.ci_support\build_all.py", line 174, in build_folders
    conda_build.api.build([recipe], config=get_config(arch, channel_urls))
  File "C:\Miniforge\lib\site-packages\conda_build\api.py", line 253, in build
    return build_tree(
  File "C:\Miniforge\lib\site-packages\conda_build\build.py", line 3799, in build_tree
    packages_from_this = build(
  File "C:\Miniforge\lib\site-packages\conda_build\build.py", line 2624, in build
    windows.build(
  File "C:\Miniforge\lib\site-packages\conda_build\windows.py", line 341, in build
    check_call_env(
  File "C:\Miniforge\lib\site-packages\conda_build\utils.py", line 450, in check_call_env
    return _func_defaulting_env_to_os_environ("call", *popenargs, **kwargs)
  File "C:\Miniforge\lib\site-packages\conda_build\utils.py", line 426, in _func_defaulting_env_to_os_environ
    raise subprocess.CalledProcessError(proc.returncode, _args)
subprocess.CalledProcessError: Command '['cmd.exe', '/d', '/c', 'conda_build.bat']' returned non-zero exit status 1.
##[error]Cmd.exe exited with code '1'.
##[section]Finishing: Build recipe
```

Seems to be failing after you removed the `export CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_LINKER=$CC` lines in commit daed7af03c86414f8575a1dc7091fd1e707bf2eb. But you've already set `- {{ compiler('rust') }}`, so I'm wondering if this `{{ compiler('m2w64_c') }}` line is the issue?

Reply:
Trying now. By the way, if you have access to push to this branch feel free to go ahead. I'm working on getting v0.6.4 of hidefix out with updated dependencies. That tends to make building easier as well.

Reply:
Nope, I'm not a conda-forge core maintainer, so don't have push permissions. But it looks like you're on the right track!

Reply:
Maybe this is related to the `ucrt` runtime, maybe we can just use compiler('cxx') on win as well.